test/Worker.Extensions.DurableTask.Analyzers.Test/Worker.Extensions.DurableTask.Analyzers.Test.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
seems the indentation here is a bit large - can we make it more conservative? 2 to 4 spaces should suffice, or just a single tab